build/lib/common.sh

```diff
@@ -181,6 +181,19 @@ function build::non-golang::gather_licenses(){
   rm -rf $project_name
 }
 
+function build::non-golang::copy_licenses(){
```

Comment:
Can we standardize this for the above method somehow?  Haven't looked too closely at the changes but could this be made so it could be called by gather_licenses after it clones/checksout the repo?

Reply:
Done